src/pages/CircleAdminPage/CircleAdminPage.tsx

```diff
@@ -370,206 +276,564 @@ export const CircleAdminPage = () => {
-            {...nominationDaysLimit}
-            helperText="(# of days)"
-            fullWidth
-            disabled={!vouching.value}
-          />
-        </div>
-        <ApeTextField
-          label="Allocation page text"
-          {...allocText}
-          multiline
-          rows={5}
-          inputProps={{
-            maxLength: 280,
-          }}
-          fullWidth
-        />
-        <div
-          className={clsx(classes.vouchingItem, !vouching.value && 'disabled')}
-        >
-          <ApeTextField
-            label="Vouching text"
-            placeholder="This is a custom note we can optionally display to users on the vouching page, with guidance on who to vouch for and how."
-            {...vouchingText}
-            multiline
-            rows={5}
-            inputProps={{
-              maxLength: 280,
-            }}
-            fullWidth
-            disabled={!vouching.value}
-          />
-        </div>
-        <ApeToggle
-          {...onlyGiverVouch}
-          className={clsx(classes.vouchingItem, !vouching && 'disabled')}
-          label="Only Givers can vouch"
-          infoTooltip={
-            <YesNoTooltip
-              yes={`Only members who are eligible to send ${
-                circle.tokenName || 'GIVE'
-              } can vouch for new members`}
-              no="Anyone in the circle can vouch for new members"
-              href={DOCS_HREF}
-              anchorText={DOCS_TEXT}
-            />
-          }
-        />
-        <ApeToggle
-          {...teamSelection}
-          label="Team Selection Enabled"
-          infoTooltip={
-            <YesNoTooltip
-              yes="Members select a team during allocation and make allocations only to that team"
-              no="Members make allocations to anyone in the circle"
-            />
-          }
-        />
-        <ApeToggle
-          {...autoOptOut}
-          label="Auto Opt Out?"
-          infoTooltip={
-            <YesNoTooltip
-              yes="If a member doesn't make allocations in an epoch, they'll be set to opt out of receiving allocations in the next epoch. They can still opt back in."
-              no="Members' opt-in/opt-out settings will not be changed automatically."
-            />
-          }
-        />
-        {isFeatureEnabled('fixed_payments') && (
-          <FormAutocomplete
-            {...fixedPaymentToken}
-            options={tokens}
-            label="Fixed Payment Token"
-            fullWidth
-          />
-        )}
-      </Box>
-      <AdminIntegrations circleId={circleId} />
-      <div className={classes.bottomContainer}>
-        <p className={classes.subTitle}>Discord Webhook</p>
-        {allowEdit && (
-          <input
-            readOnly={!allowEdit}
-            className={classes.input}
-            {...discordWebhook}
-          />
-        )}
-        <div className={classes.webhookButtonContainer}>
-          {!allowEdit && (
+    <>
```

Comment:
this fragment tag is unnecessary because there's only one child tag inside it

Reply:
done